src/test/resources/uk/gov/justice/digital/hmpps/hmppstier/integration/bdd/mandateForChange.9.5.feature

```diff
@@ -6,13 +6,13 @@ Feature: Mandate for change
     Given an offender with a current sentence of type 'SC'
     And an OGRS score of "50"%
     When a tier is calculated
-    Then there is a mandate for change and a change level of "1" is returned for "5" points
+    Then there is a mandate for change and a change level of 1 is returned and 5 points are scored
```

Comment:
BDD picky, but should we not just be asserting <> level 1 here  as 'There is mandate for change'

Reply:
```
Given an offender with a current sentence of type 'SC'
When a tier is calculated
Then there is a mandate for change
```
'Then there is mandate for change: changeLevel != 0